client/src/components/Viewer/components/GetFiles/components/RenderFiles/components/OpenSeaDragon/OpenSeadragonViewer.js

```diff
@@ -1,103 +1,75 @@
 import React, { useEffect } from 'react';
 import OpenSeadragon from 'openseadragon';
 import './OpenSeadragon.css';
+import { config } from "../../../../../../../Config/config";
 
-function OpenSeadragonViewer({ imageUrl, info, format, outer }) {
-  let viewer;
-  useEffect(() => {
-    if (format === 'png' || format === 'jpeg') {
-      viewer = OpenSeadragon({
-        id: 'openseadragon-viewer',
-        prefixUrl:
-          ' https://cdn.jsdelivr.net/npm/openseadragon@2.4/build/openseadragon/images/',
-        tileSources: {
-          type: 'image',
-          url: imageUrl,
-          buildPyramid: false,
-        },
-        animationTime: 0.5,
-        blendTime: 0.1,
-        constrainDuringPan: true,
-        maxZoomPixelRatio: 2,
-        minZoomLevel: 1,
-        visibilityRatio: 1,
-        zoomPerScroll: 2,
-        showNavigator: true,
-        ajaxWithCredentials: true,
-        sequenceMode: true,
-        crossOriginPolicy: 'Anonymous',
-      });
-    } else {
-      viewer = OpenSeadragon({
-        id: 'openseadragon-viewer',
-        prefixUrl:
-          ' https://cdn.jsdelivr.net/npm/openseadragon@2.4/build/openseadragon/images/',
-        tileSources: {
-          width: 28480,
-          height: 28760,
-          tileSize: 512,
-          tileOverlap: 0,
-          getTileUrl: function (level, x, y) {
-            if (info[level + '/' + x + '_' + y] !== undefined) {
-              let signature = info[level + '/' + x + '_' + y][0];
-              let date = info[level + '/' + x + '_' + y][1];
-              let credential = info[level + '/' + x + '_' + y][2];
-              let startLink = outer.split('_files')[0];
+function OpenSeadragonViewer({imageName,imageUrl,info,format,outer,email}) {
+        let viewer;
+        useEffect(() => {
+          if(format == 'png' || format == 'jpeg')
```

Comment:
use "===" here
+ better do this

create a 

```js
const SIMPLE_IMAGE_FORMATS  = ['png','jpeg'];

if(SIMPLE_IMAGE_FORMATS.includes(format)){...}
```

and move this constants to components/OpenSeaDragon/constants.js


This is because we might need this constant in multiple places

Reply:
added